tests/framework/CodeInterfaceTests/test_Prescient_code_interface_fake.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" ?>
+<Simulation>
+  <TestInfo>
+    <name>framework/CodeInterfaceTests/PrescientInterface</name>
+    <author>@cogljj</author>
+    <created>2020-09-02</created>
+    <revisions>
+    </revisions>
+    <classesTested>CodeInterfaces.PrescientCodeInterface</classesTested>
+    <description>
+      Test the Prescient Code Interface.
+    </description>
+  </TestInfo>
+
+  <RunInfo>
+    <WorkingDir>PrescientInterfaceTest</WorkingDir>
+    <Sequence>runPrescientFake</Sequence>
+    <batchSize>1</batchSize>
+  </RunInfo>
+
+  <Files>
+    <Input name="simulate" type="PrescientRunnerInput">simulate_with_network_deterministic_day.txt</Input>
+    <Input name="structure" type="PrescientInput" subDirectory="deterministic_with_network_scenarios_day/pyspdir_twostage/2020-07-10/">ScenarioStructure.dat</Input>
+    <Input name="scenario_1" type="PrescientInput" subDirectory="deterministic_with_network_scenarios_day/pyspdir_twostage/2020-07-10/">Scenario_1.dat</Input>
+    <Input name="actuals" type="PrescientInput" subDirectory="deterministic_with_network_scenarios_day/pyspdir_twostage/2020-07-10/">Scenario_actuals.dat</Input>
+    <Input name="forcasts" type="PrescientInput" subDirectory="deterministic_with_network_scenarios_day/pyspdir_twostage/2020-07-10/">Scenario_forecasts.dat</Input>
+    <Input name="scenarios" type="PrescientInput" subDirectory="deterministic_with_network_scenarios_day/pyspdir_twostage/2020-07-10/">scenarios.csv</Input>
```

Comment:
are all of these files needed for a Prescient calculation? If so, how can be sure that all the files have been inputted by the user (since all of them have the same type "PrescientInput")?

Reply:
Yes, all these files are needed, and for what it is worth, this is one chosen to have a small number of files.  Possibly we need to have some way to copy over a directory for Prescient.